contracts/TokenBalanceOracle.sol

```diff
@@ -11,6 +11,9 @@ contract TokenBalanceOracle is AragonApp, IACLOracle {
     bytes32 public constant SET_MIN_BALANCE_ROLE = keccak256("SET_MIN_BALANCE_ROLE");
 
     string private constant ERROR_TOKEN_NOT_CONTRACT = "ORACLE_TOKEN_NOT_CONTRACT";
+    string private constant ERROR_SENDER_MISSING = "ORACLE_SENDER_MISSING";
+    string private constant ERROR_SENDER_TOO_BIG = "ORACLE_SENDER_TOO_BIG";
+    string private constant ERROR_SENDER_ZERO = "ORACLE_SENDER_ZERO";
```

Comment:
Consider prefixing these errors with "TOKEN_BALANCE" eg `string private constant ERROR_SENDER_ZERO = "TOKEN_BALANCE_ORACLE_SENDER_ZERO";` for consistency with the other apps.